src/github.com/getlantern/flashlight/main/main.go

```diff
@@ -53,7 +54,12 @@ func init() {
 }
 
 func logPanic(msg string) {
-	cfg, err := config.Init(flashlight.PackageVersion, *configdir, *stickyConfig, flagsAsMap())
+	// Request the config via either chained servers or direct fronted servers.
+	// We have to do the full version here because we're actually trying to
+	// submit this panic to our logs.
+	cf := util.NewChainedAndFronted(client.Addr)
+	configFetcher := config.NewFetcher(settings.GetUserID, settings.GetToken, cf)
```

Comment:
Lantern does require the UI to sign-up for pro, with that you generate an appropriate yaml file and with that yaml file you can run headless from that point on, if you want.